packages/vite/src/node/utils.ts

```diff
@@ -571,6 +571,23 @@ export function copyDir(srcDir: string, destDir: string): void {
   }
 }
 
+/**
+ * Wrapper to prevent `fs.realpathSync.native` errors in Windows virtual and RAM disks
+ * that bypass the Volume Mount Manager, in programs such as imDisk
+ * @param path - a path to a file
+ * @returns {string}
+ */
+function wrapRealpathSyncNative(path: string): string {
+  if (typeof fs.realpathSync.native === 'function') {
+    try {
+      return fs.realpathSync.native(path)
+    } catch {
+      // Ignore errors
```

Comment:
Is there a specific error we could ignore here? If not, we end up calling `realPathSync` twice for every error.